### PR TITLE
Add cron job to deactivate expired cards

### PR DESCRIPTION
**Pull request by IKarbowiak**
Run celery task every day at midnight.

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-4431 Add cron job to deactivate expired gift cards](https://app.clickup.com/t/2549495/SALEOR-4431)

File: saleor/giftcard/tasks.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+from .events import gift_cards_deactivated_event
+from .models import GiftCard
+
+task_logger = get_task_logger(__name__)
+
+
+@app.task
+def deactivate_expired_cards_task():
+    today = timezone.now().date()
+    gift_cards = GiftCard.objects.filter(expiry_date__lt=today)
```
Comment (korycins) [inline saleor/giftcard/tasks.py:14]: Shouldn't be limited to `is_ative=True` here?

Comment (IKarbowiak) [reply saleor/giftcard/tasks.py:14]: True. Thanks!

Comment (korycins) [reply saleor/giftcard/tasks.py:14]: 🚔  😄

**Review by fowczarek — APPROVED** [review]

**Review by korycins — APPROVED** [review]

**Pull request merged**